policyengine_api/api.py

```diff
@@ -57,7 +58,7 @@
 
 app.route("/", methods=["GET"])(get_home)
 
-app.route("/<country_id>/metadata", methods=["GET"])(get_metadata)
+app.register_blueprint(metadata_bp)
```

Comment:
I believe this is missing a URL prefix setting
```suggestion
app.register_blueprint(metadata_bp, url_prefix="/<country_id>/metadata")
```

Reply:
See the comment I made on [your PR](https://github.com/PolicyEngine/policyengine-api/pull/2025) and let me know which way you want to go. I dislike both options for at least one reason.. so you pick. I think the real solution is to flatten the hierarchy (full details in that review).

Reply:
Pursuant to [this discussion](https://github.com/PolicyEngine/policyengine-api/pull/2025), I do now think the code you have is the way we want to go. Resolving.